neasqc_wp61/data/data_processing/use_beta_1.py

```diff
@@ -0,0 +1,83 @@
+import sys
+import os
+import argparse
+import time 
+import json
+current_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(current_path + "/../../models/quantum/beta_1/")
+from beta_1 import QuantumKNearestNeighbours as qkn
```

Comment:
/!\ Here in the set_up_exp PR, the beta folder + beta files are renamed. It's not the case for the new general_quantum_distance PR.

Reply:
When merging the branch general_quantum_distance, I assume that a merging conflict will happen regarding the names of the files. We can solve it by keeping the names of the set_up_experiments branch, and this will be fixed.